client/StatusTable/components/WeekRow.js

```diff
@@ -0,0 +1,26 @@
+import React, {useCallback, useState} from "react";
+import DayRow from './DayRow'
+import classNames from './scss/WeekRow.module.scss'
+import getClassNames from "../../utilities/getClassnames";
+
+const WeekRow = ({workweek}) => {
+    const [isCollapsed, setCollapse] = useState(false);
+    const handleWeekClick = useCallback( () => {
+        setCollapse(!isCollapsed);
+    }, [isCollapsed]);
+
+    const firstChildClasses = getClassNames({[classNames['week-row__cell']]: true, [classNames['week-row__cell_name-of-day']]: true});
```

Comment:
it is easier to read long string like this if they are split on several strings :)